File: TreasurersApp/ClientApp/src/app/models/Address.ts
export class Address {
  id: number = null;
  addressLine1: string = null;
  addressLine2: string = null;
  addressLine3: string = null;
  city: string = null;
  state: string = null;
  postalCode: string = null;
  createdBy: string = null;
  createdDate: Date = null;
  lastModifiedBy: string = null;
  lastModifiedDate: Date = null;

  constructor(id: number,
    addressLine1: string,
    addressLine2: string,
    addressLine3: string,
    city: string,
    state: string,
    postalCode: string,
    createdBy: string,
    createdDate: Date,
    lastModifiedBy: string,
    lastModifiedDate: Date) {
    this.id = id;
    this.addressLine1 = addressLine1;
    this.addressLine2 = addressLine2;
    this.addressLine3 = addressLine3;
    this.city = city;
    this.state = state;
    this.postalCode = postalCode;
    this.createdBy = createdBy;
    this.createdDate = createdDate;
    this.lastModifiedBy = lastModifiedBy;
    this.lastModifiedDate = lastModifiedDate;
  }

  asAddressString() {
    return this.addressLine1 + ', ' + this.city + ', ' + this.state + ' ' + this.postalCode;
  }
}

File: TreasurersApp/Models/Address.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace TreasurersApp.Models
{
    public partial class Address
    {
        [JsonProperty("id")]
        public int AddressId { get; set; }

        [JsonProperty("addressLine1")]
        public string AddressLine1 { get; set; }

        [JsonProperty("addressLine2")]
        public string AddressLine2 { get; set; }

        [JsonProperty("addressLine3")]
        public string AddressLine3 { get; set; }

        [JsonProperty("city")]
        public string City { get; set; }

        [JsonProperty("state")]
        public string State { get; set; }

        [JsonProperty("postalCode")]
        public string PostalCode { get; set; }

        [JsonProperty("createdBy")]
        public System.Guid CreatedBy { get; set; } // CreatedBy

        [JsonProperty("createdDate")]
        public System.DateTime CreatedDate { get; set; } // CreatedDate

        [JsonProperty("lastModifiedBy")]
        public System.Guid LastModifiedBy { get; set; } // LastModifiedBy

        [JsonProperty("lastModifiedDate")]
        public System.DateTime LastModifiedDate { get; set; } // LastModifiedDate

        /// <summary>
        /// Child ContributorAddresses where [ContributorAddress].[AddressID] point to this entity (FK_ContributorAddress_Address)
        /// </summary>
        public System.Collections.Generic.ICollection<ContributorAddress> ContributorAddresses { get; set; } // ContributorAddress.FK_ContributorAddress_Address

        public Address()
        {
            ContributorAddresses = new System.Collections.Generic.List<ContributorAddress>();
        }

    }
}
